---a/net/minecraft/item/ItemFishFood.java
+++b/net/minecraft/item/ItemFishFood.java
@@ -1,51 +1,42 @@
 package net.minecraft.item;
 
-import com.google.common.collect.Maps;
-import java.util.List;
 import java.util.Map;
-import net.minecraft.creativetab.CreativeTabs;
+
 import net.minecraft.entity.player.EntityPlayer;
 import net.minecraft.potion.Potion;
 import net.minecraft.potion.PotionEffect;
 import net.minecraft.potion.PotionHelper;
 import net.minecraft.world.World;
-import net.minecraftforge.fml.relauncher.Side;
-import net.minecraftforge.fml.relauncher.SideOnly;
 
-public class ItemFishFood extends ItemFood
-{
+import com.google.common.collect.Maps;
+
+public class ItemFishFood extends ItemFood {
+
     private final boolean cooked;
     private static final String __OBFID = "CL_00000032";
 
-    public ItemFishFood(boolean cooked)
-    {
+    public ItemFishFood(boolean cooked) {
         super(0, 0.0F, false);
         this.cooked = cooked;
     }
 
-    public int getHealAmount(ItemStack stack)
-    {
+    public int getHealAmount(ItemStack stack) {
         ItemFishFood.FishType fishtype = ItemFishFood.FishType.byItemStack(stack);
         return this.cooked && fishtype.canCook() ? fishtype.getCookedHealAmount() : fishtype.getUncookedHealAmount();
     }
 
-    public float getSaturationModifier(ItemStack stack)
-    {
+    public float getSaturationModifier(ItemStack stack) {
         ItemFishFood.FishType fishtype = ItemFishFood.FishType.byItemStack(stack);
         return this.cooked && fishtype.canCook() ? fishtype.getCookedSaturationModifier() : fishtype.getUncookedSaturationModifier();
     }
 
-    public String getPotionEffect(ItemStack stack)
-    {
+    public String getPotionEffect(ItemStack stack) {
         return ItemFishFood.FishType.byItemStack(stack) == ItemFishFood.FishType.PUFFERFISH ? PotionHelper.pufferfishEffect : null;
     }
 
-    protected void onFoodEaten(ItemStack stack, World worldIn, EntityPlayer player)
-    {
+    protected void onFoodEaten(ItemStack stack, World worldIn, EntityPlayer player) {
         ItemFishFood.FishType fishtype = ItemFishFood.FishType.byItemStack(stack);
 
-        if (fishtype == ItemFishFood.FishType.PUFFERFISH)
-        {
+        if (fishtype == ItemFishFood.FishType.PUFFERFISH) {
             player.addPotionEffect(new PotionEffect(Potion.poison.id, 1200, 3));
             player.addPotionEffect(new PotionEffect(Potion.hunger.id, 300, 2));
             player.addPotionEffect(new PotionEffect(Potion.confusion.id, 300, 1));
@@ -54,48 +54,12 @@
         super.onFoodEaten(stack, worldIn, player);
     }
 
-    @SideOnly(Side.CLIENT)
-    public void getSubItems(Item itemIn, CreativeTabs tab, List subItems)
-    {
-        ItemFishFood.FishType[] afishtype = ItemFishFood.FishType.values();
-        int i = afishtype.length;
-
-        for (int j = 0; j < i; ++j)
-        {
-            ItemFishFood.FishType fishtype = afishtype[j];
-
-            if (!this.cooked || fishtype.canCook())
-            {
-                subItems.add(new ItemStack(this, 1, fishtype.getMetadata()));
-            }
-        }
-    }
-
-    public String getUnlocalizedName(ItemStack stack)
-    {
+    public String getUnlocalizedName(ItemStack stack) {
         ItemFishFood.FishType fishtype = ItemFishFood.FishType.byItemStack(stack);
         return this.getUnlocalizedName() + "." + fishtype.getUnlocalizedName() + "." + (this.cooked && fishtype.canCook() ? "cooked" : "raw");
     }
 
-    public static enum FishType
-    {
+    public static enum FishType {
         COD(0, "cod", 2, 0.1F, 5, 0.6F),
         SALMON(1, "salmon", 2, 0.1F, 6, 0.8F),
         CLOWNFISH(2, "clownfish", 1, 0.1F),
@@ -94,8 +94,7 @@
 
         private static final String __OBFID = "CL_00000033";
 
-        private FishType(int meta, String unlocalizedName, int uncookedHeal, float uncookedSaturation, int cookedHeal, float cookedSaturation)
-        {
+        private FishType(int meta, String unlocalizedName, int uncookedHeal, float uncookedSaturation, int cookedHeal, float cookedSaturation) {
             this.meta = meta;
             this.unlocalizedName = unlocalizedName;
             this.uncookedHealAmount = uncookedHeal;
@@ -105,8 +105,7 @@
             this.cookable = true;
         }
 
-        private FishType(int meta, String unlocalizedName, int uncookedHeal, float uncookedSaturation)
-        {
+        private FishType(int meta, String unlocalizedName, int uncookedHeal, float uncookedSaturation) {
             this.meta = meta;
             this.unlocalizedName = unlocalizedName;
             this.uncookedHealAmount = uncookedHeal;
@@ -116,63 +116,51 @@
             this.cookable = false;
         }
 
-        public int getMetadata()
-        {
+        public int getMetadata() {
             return this.meta;
         }
 
-        public String getUnlocalizedName()
-        {
+        public String getUnlocalizedName() {
             return this.unlocalizedName;
         }
 
-        public int getUncookedHealAmount()
-        {
+        public int getUncookedHealAmount() {
             return this.uncookedHealAmount;
         }
 
-        public float getUncookedSaturationModifier()
-        {
+        public float getUncookedSaturationModifier() {
             return this.uncookedSaturationModifier;
         }
 
-        public int getCookedHealAmount()
-        {
+        public int getCookedHealAmount() {
             return this.cookedHealAmount;
         }
 
-        public float getCookedSaturationModifier()
-        {
+        public float getCookedSaturationModifier() {
             return this.cookedSaturationModifier;
         }
 
-        public boolean canCook()
-        {
+        public boolean canCook() {
             return this.cookable;
         }
 
-        public static ItemFishFood.FishType byMetadata(int meta)
-        {
-            ItemFishFood.FishType fishtype = (ItemFishFood.FishType)META_LOOKUP.get(Integer.valueOf(meta));
+        public static ItemFishFood.FishType byMetadata(int meta) {
+            ItemFishFood.FishType fishtype = (ItemFishFood.FishType) META_LOOKUP.get(Integer.valueOf(meta));
             return fishtype == null ? COD : fishtype;
         }
 
-        public static ItemFishFood.FishType byItemStack(ItemStack stack)
-        {
+        public static ItemFishFood.FishType byItemStack(ItemStack stack) {
             return stack.getItem() instanceof ItemFishFood ? byMetadata(stack.getMetadata()) : COD;
         }
 
-        static
-        {
+        static {
             ItemFishFood.FishType[] var0 = values();
             int var1 = var0.length;
 
-            for (int var2 = 0; var2 < var1; ++var2)
-            {
+            for (int var2 = 0; var2 < var1; ++var2) {
                 ItemFishFood.FishType var3 = var0[var2];
                 META_LOOKUP.put(Integer.valueOf(var3.getMetadata()), var3);
             }
         }
     }
-}
+}
